### PR TITLE
Add RBAC guidance to users and teams documentation 

### PR DESCRIPTION
**Pull request by alyssawada**
# What this PR does
RBAC for OnCall will be GA with Grafana 9.4 release.
Update [this doc](https://grafana.com/docs/oncall/latest/configure-user-settings/) to include details about RBAC for OnCall and link to RBAC-specific docs.

## Which issue(s) this PR fixes
Issue #1399 

## Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated

### CODE REVIEW

**Comment by joeyorlando** [comment]
@alyssawada overall this LGTM. I think you have some extra commits in here causing the changes in the `/helm` directory. once that's sorted out I'll go ahead and approve (unless you meant to make those helm related changes?)

File: helm/oncall/Chart.yaml [hunk 1]
```diff
@@ -1,19 +1,9 @@
...
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.2
-
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-# It is recommended to use it with quotes.
-appVersion: "v1.1.5"
+version: 1.1.29
```
Comment (iskhakov) [inline helm/oncall/Chart.yaml:5]: I believe this change can be removed from this PR

**Review by iskhakov — CHANGES_REQUESTED** [review]
please remove the changes to the following files helm/oncall/Chart.lock, helm/oncall/Chart.lock, helm/oncall/Chart.lock

**Comment by joeyorlando** [comment]
bump (cc: @alyssawada)

**Review by joeyorlando — APPROVED** [review]

**Pull request merged**